pkgs/development/python-modules/pari-jupyter/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cython
+, pari
+, readline
+, ipykernel
+}:
+
+buildPythonPackage rec {
+  pname = "pari-jupyter";
+  version = "1.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5fa773d725572ebeb2de7e8db23bc73df40bfeb8fae55e31125d4bb78dc992c3";
+  };
+
+  buildInputs = [ readline pari ];
+
+  propagatedBuildInputs = [
+    ipykernel
+    cython
+  ];
+
+  doCheck = false;
```

Comment:
Please comment that the repo has no tests presumable and add a pythonImportsCheck.

Reply:
I just dropped this commit. We can re-add it later if needed.